components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -67,10 +67,11 @@ export const PrebuildDetailPage: FC = () => {
     const triggeredString = useMemo(() => formatDate(triggeredDate), [triggeredDate]);
 
     useEffect(() => {
+        setLogNotFound(false);
         watchPrebuild(prebuildId, (prebuild) => {
             setCurrentPrebuild(prebuild);
         });
-    }, [prebuildId, setCurrentPrebuild]);
+    }, [prebuildId]);
```

Comment:
Only reactive values should be in the dependency arrays. Not sure if it can possibly cause unwanted renders but removed just to make sure

From an example:
https://react.dev/learn/removing-effect-dependencies#are-you-reading-some-state-to-calculate-the-next-state